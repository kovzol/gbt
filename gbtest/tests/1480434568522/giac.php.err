// Using locale /usr/share/locale/
// C
// /usr/share/locale/
// giac
// UTF-8
// Maximum number of parallel threads 8
Help file /usr/share/giac/doc/local/aide_cas not found
Added 0 synonyms

// Groebner basis computation time 0.000583 Memory 0.022956M
// Time 0
0.04user 0.00system 0:00.05elapsed 98%CPU (0avgtext+0avgdata 24812maxresident)k
0inputs+16outputs (0major+1549minor)pagefaults 0swaps
