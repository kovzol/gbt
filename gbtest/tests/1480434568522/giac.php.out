Welcome to giac readline interface
(c) 2001,2016 B. Parisse & others
Homepage http://www-fourier.ujf-grenoble.fr/~parisse/giac.html
Released under the GPL license 3.0 or above
See http://www.gnu.org for license details
May contain BSD licensed software parts (lapack, atlas, tinymt)
-------------------------------------------------
Press CTRL and D simultaneously to finish session
Type ?commandname for help
0>> gbasis([-1*v11,-1*v9,-1+v12,v10,-1+v7,v6,v8,v5,-1*v11*v10+v12*v9+v11*v4+-<+v12,v10,-1+v7,v6,v8,v5,-1*v11*v10+v12*v9+v11*v4+-1                         *v9*v4+-1*v12*v3+v10*v3,-<*v11*v10+v12*v9+v11*v4+-1*v9*v4+-1*v12*v3+v10*v3,-1                         *v8+v6+-1*v7*v6+v8*v5,-1*<v9*v4+-1*v12*v3+v10*v3,-1*v8+v6+-1*v7*v6+v8*v5,-1*v                         7*v6+v8*v5+v7*v4+-1*v5*v4<v8+v6+-1*v7*v6+v8*v5,-1*v7*v6+v8*v5+v7*v4+-1*v5*v4+                         -1*v8*v3+v6*v3,-1+v19^2+-<*v6+v8*v5+v7*v4+-1*v5*v4+-1*v8*v3+v6*v3,-1+v19^2+-1                         *v2^2+2*v1+-1*v1^2,v21^4+<1*v8*v3+v6*v3,-1+v19^2+-1*v2^2+2*v1+-1*v1^2,v21^4+-                         2*v21^2*v20^2+v20^4+-2*v2<v2^2+2*v1+-1*v1^2,v21^4+-2*v21^2*v20^2+v20^4+-2*v21                         ^2*v19^2+-2*v20^2*v19^2+v<*v21^2*v20^2+v20^4+-2*v21^2*v19^2+-2*v20^2*v19^2+v1                         9^4,-1*v16^2+2*v15+-1*v15<2*v19^2+-2*v20^2*v19^2+v19^4,-1*v16^2+2*v15+-1*v15^                         2+v4^2+-2*v3+v3^2,v20^2+-<^4,-1*v16^2+2*v15+-1*v15^2+v4^2+-2*v3+v3^2,v20^2+-1                         *v4^2+-1*v3^2+2*v4*v2+-1*<+v4^2+-2*v3+v3^2,v20^2+-1*v4^2+-1*v3^2+2*v4*v2+-1*v                         2^2+2*v3*v1+-1*v1^2,1+-1*<v4^2+-1*v3^2+2*v4*v2+-1*v2^2+2*v3*v1+-1*v1^2,1+-1*v                         16^2+-1*v15^2+2*v16*v4+-2<^2+2*v3*v1+-1*v1^2,1+-1*v16^2+-1*v15^2+2*v16*v4+-2*                         v3+2*v15*v3,v21^2+-1*v16^<6^2+-1*v15^2+2*v16*v4+-2*v3+2*v15*v3,v21^2+-1*v16^2                         +-1*v15^2+2*v16*v2+-1*v2^<3+2*v15*v3,v21^2+-1*v16^2+-1*v15^2+2*v16*v2+-1*v2^2                         +2*v15*v1+-1*v1^2],[v20,v<-1*v15^2+2*v16*v2+-1*v2^2+2*v15*v1+-1*v1^2],[v20,v2                         1,v19,v16,v15,v12,v11,v10<2*v15*v1+-1*v1^2],[v20,v21,v19,v16,v15,v12,v11,v10,                         v9,v8,v7,v6,v5,v4,v3,v1,v<,v19,v16,v15,v12,v11,v10,v9,v8,v7,v6,v5,v4,v3,v1,v2                         ],revlex)
[3*v1^4-4*v16*v1^2*v2+6*v1^2*v2^2-4*v16*v2^3+3*v2^4-6*v1^3+4*v16*v1*v2-6*v1*v2^2+3*v1^2+v2^2,v20^2-v1^2-v2^2,v21^2-v1^2+2*v16*v2-v2^2+v1-1,v19^2-v1^2-v2^2+2*v1-1,4*v16^2-3,2*v15-1,v12-1,v11,v10,v9,v8,v7-1,v6,v5,v4,v3]
1>> 